org.kde.klevernotes.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "org.kde.klevernotes",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
```

Comment:
6.7 is the latest, any reason you aren't using it ?

Reply:
You are right, I did not noticed it, I will bump the version.